includes/Main.php

```diff
@@ -1286,9 +1311,10 @@ public function log_document_creation_to_order_notes( $document, $trigger ) {
 	 */
 	public function log_document_deletion_to_order_notes( object $document ): void {
```

Comment:
Maybe we can make this more modular and pass the trigger too?

Reply:
In this case, we need to add a `$trigger` parameter to the `delete()` method to pass it to the log method. I'm not sure if that's a good idea. What do you think?

Reply:
Got it. Let’s leave it as it is for now.